ComputerVision/SDK/AnalyzeImage/AnalyzeImage.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
For new samples, I recommend to put them in https://github.com/Azure-Samples/cognitive-services-dotnet-sdk-samples/tree/master/samples so that it can be auto tested.

Strongly recommended to void future re-work, but not a blocking issue.